doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+## Mattermost Config Settings
+___
+Config settings allow system administrators to customize their teams Mattermost experience. Warning: system administrators should be cautious to only grant admin privileges to team members who understand the critical nature of the settings they are adjusting.
+
+
+## System Console Settings:
+These settings can be modified from the System Console user interface or config.json.
+
+### Service Settings
+___
+General settings to configure the listening address, login security, testing, webhooks and service integration of Mattermost. 
+
+### System
+
+```"ListenAddress": ":8065"```
+*The IP address to listen and the port which to bind. Entering ":8065" will bind to all interfaces or you can choose one like "127.0.0.1:8065". Changing this will require a server restart before taking effect.*
+
+```"MaximumLoginAttempts": 10```
+*Failed login attempts allowed before user is locked out and required to reset password via email.*
+
+```"SegmentDeveloperKey": ""```
+*For users running SaaS services, sign up for a key at Segment.com to track metrics.*
+
+```"GoogleDeveloperKey": ""```
+*Set this key to enable embedding of YouTube video previews based on hyperlinks appearing in messages or comments. Instructions to obtain a key available at https://www.youtube.com/watch?v=Im69kzhpR3I. Leaving field blank disables the automatic generation of YouTube video previews from links.*
+
+```"EnableTesting": false```
+*"true": /loadtest slash command is enabled to load test accounts and test data.*
+
+### Webhooks
+
+```"EnableIncomingWebhooks": true```
+*Developers building integrations can create webhook URLs for channels and private groups. Please see http://mattermost.org/webhooks to learn about creating webhooks, view samples, and to let the community know about integrations you have built. "true": Incoming webhooks will be allowed and all posts from webhooks will be labelled with a BOT tag. To manage incoming webhooks, go to Account Settings -> Integrations. The webhook URLs created in Account Settings can be used by external applications to create posts in any channels or private groups you have access to; “false”: The Integrations tab of Account Settings is hidden and incoming webhooks are disabled*
+
+```"EnablePostUsernameOverride": false```
+*"true": Webhooks will be allowed to change the username they are posting as; “false”: Webhooks can only post as the username they were set up with. See http://mattermost.org/webhooks for more details.*
```

Comment:
Might be a good idea to mention something like:

"Note that users may be able to perform [phishing attacks](https://en.wikipedia.org/wiki/Phishing) by attempting to impersonate other users using this feature. To combat this a BOT tag appears next to all posts from a webhook. Enable at your own risk."